projects/plugins/jetpack/extensions/shared/external-media/index.js

```diff
@@ -1,6 +1,7 @@
 import { isCurrentUserConnected } from '@automattic/jetpack-shared-extension-utils';
 import { useBlockEditContext } from '@wordpress/block-editor';
 import { addFilter } from '@wordpress/hooks';
+import { SOURCE_JETPACK_APP_MEDIA } from './constants'; // Move this import statement above the import statement for './media-button'
```

Comment:
I don't quite understand the comment here. Can it be removed?

```suggestion
import { SOURCE_JETPACK_APP_MEDIA } from './constants';
```